### ISSUE TITLE
Text link not accessible

### ISSUE THREAD

**Issue by weboverhauls** [cutestrap/cutestrap#25]
The text link is not fully accessible since it relies on color alone to denote that it's a link.

Suggestion 1: return the underline. (highly recommended)

Suggestion 2: provide more visual affordance that the standalone text is a link such as adding an icon.

WCAG references: 
http://www.w3.org/TR/2008/REC-WCAG20-20081211/#visual-audio-contrast-without-color
http://www.w3.org/TR/WCAG20-TECHS/F73.html

PS:
Since this is a link, strange that it's listed under Buttons and the class names say button.

![image](https://cloud.githubusercontent.com/assets/473400/15724081/825a078a-2813-11e6-9ecd-b83a190de62f.png)

**Comment by thisconnect** [comment]
> Suggestion 1: return the underline. (highly recommended)

I agree especially because all other links in cutestrap have the underlines 

> Suggestion 2: provide more visual affordance that the standalone text is a link such as adding an icon.

One should be free to choose an icon set, so I hope cutestrap does **NOT** include icons

> Since this is a link, strange that it's listed under Buttons and the class names say button.

[Other](http://getbootstrap.com/css/#buttons-options) frameworks use button style link as well.

**Comment by tylerchilds** [comment]
Definitely will add the underline, that was just an oversight of mine when developing.

Won't be adding icons, but I might build a light weight icon set at some point in the future to be used in conjunction with cutestrap.

**Comment by patrickhlauke** [comment]
> Since this is a link, strange that it's listed under Buttons and the class names say button.

I'd suggest that, to make it clearer, the example should add the `btn--link` class to a `<button>`, and the `<a>` should get another class like `btn` instead...

**Comment by patrickhlauke** [comment]
...something like

```
<button>Default</button>
<button class="btn--secondary">Secondary</button>
<button class="btn--link">Link</button>
<a class="btn" href="#">Link styled as button</a>
```

> ![cute-btns](https://cloud.githubusercontent.com/assets/895831/17023801/17c0d5d2-4f4d-11e6-9d69-42c5ef6505fa.png)
